### ISSUE TITLE
batch=32

### ISSUE THREAD

**Issue by LDZ233-ldz** [HobbitLong/SupContrast#72]
Setting the batch smaller, can the experiment achieve the same performance? Due to my graphics card memory is relatively small

**Comment by bomtorazek** [comment]
No, you can check figure 4 in the paper. The performance decreases as you reduce the batch size like the case for using the cross-entropy loss.

**Comment by dvdsosa** [comment]
@LDZ233-ldz I think you can succeed by lowering the LR in the same proportion as the BS.
